src/Uno.UI/UI/Xaml/Controls/Image/Image.Android.cs

```diff
@@ -262,29 +272,32 @@ private void TryOpenImage()
 				return;
 			}
 
-			// If the ImageSource has the UseTargetSize set, the image 
-			// must not be loaded until the first layout has been done.
-			if (Source != null && Source.UseTargetSize && !IsLoaded)
+			if (Source != null && Source.UseTargetSize)
 			{
-				if (this.Log().IsEnabled(Microsoft.Extensions.Logging.LogLevel.Debug))
+				// If the ImageSource has the UseTargetSize set, the image 
+				// must not be loaded until the first layout has been done.
+				if (!IsLoaded)
```

Comment:
Why was the condition removed ?

Reply:
It's not removed, it's just separated into 2 `if`s